src/aura/STG_CMP_Tabs/STG_CMP_TabsHelper.js

```diff
@@ -232,6 +231,18 @@
         saveAction.setCallback(this, function(response) {
             if(response.getState() === "SUCCESS") {
                 component.set("v.isView", true);
+                var tst = component.find("successToast");
+                $A.util.removeClass(tst, "slds-hide");
+                $A.util.addClass(tst, "slds-show");
+                
+                //show manual toast
+                window.setTimeout(
+                    $A.getCallback(function() {
+                        $A.util.removeClass(tst, "slds-show");
+                        $A.util.addClass(tst, "slds-hide");    
+                    }), 2500
```

Comment:
@lifewithryan I wonder if this is too fast? The default is 5 seconds, maybe we should just stick with that: https://developer.salesforce.com/docs/component-library/bundle/force:showToast/specification

Reply:
I tried 5, 3, and 2.5 -- 5 just seemed like an eternity to me. 3 was better but for me still felt long. Ya wanna save and get on with it. Maybe we go in the middle with 3750?

Reply:
Changed to 3.75 seconds -- it still feels long to me but might make it easier to robot test? If we need to go to 5, I can go to 5, but gross... :)